api/src/api/users.ts

```diff
@@ -53,6 +54,35 @@ router.get(
   }),
 );
 
+// Gets all pitches for a specific User
```

Comment:
Actually, on second thought this might not be necessary, since we're already storing the ids of the pitches that users have claimed and submitted - we can just access them directly through the User object. TLDR you can remove this endpoint 😅